evaluate/eval.py

```diff
@@ -148,13 +151,25 @@ def compute_correct_repairs(self):
         self.correct_repairs = correct_repairs
 
     def compute_recall(self):
-        return self.correct_repairs / self.total_errors
+        try:
+            return self.correct_repairs / self.total_errors
+        except Exception as e:
```

Comment:
what could go wrong here? Division by zero?
If so, please use `except ZeroDivisionError as e:`  instead. Or better, check that `self.total_errors` is not zero first and if it is zero, handle it by returning 0 or 1 or whatever you see fit in that case

Reply:
Same for the rest of the compute quality methods

Reply:
Yes, division by zero. I changed them to the latter.